src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -4243,17 +4243,30 @@ namespace PackagedTerminalHeatPump {
 
             //   set fluid-side hardware limits
             if (PTUnit(PTUnitNum).HeatCoilFluidInletNode > 0) {
-                // If water coil max water flow rate is autosized, simulate once in order to mine max water flow rate
+                // If coil max fluid flow rate is autosized, simulate once in order to mine max flow rate
                 if (PTUnit(PTUnitNum).MaxHeatCoilFluidFlow == AutoSize) {
-                    SimulateWaterCoilComponents(PTUnit(PTUnitNum).ACHeatCoilName, FirstHVACIteration, PTUnit(PTUnitNum).ACHeatCoilIndex);
-                    CoilMaxVolFlowRate = GetCoilMaxWaterFlowRate("Coil:Heating:Water", PTUnit(PTUnitNum).ACHeatCoilName, ErrorsFound);
-                    if (CoilMaxVolFlowRate != AutoSize) {
-
-                        rho = GetDensityGlycol(PlantLoop(PTUnit(PTUnitNum).HeatCoilLoopNum).FluidName,
-                                               DataGlobals::HWInitConvTemp,
-                                               PlantLoop(PTUnit(PTUnitNum).HeatCoilLoopNum).FluidIndex,
-                                               RoutineNameSpace);
-                        PTUnit(PTUnitNum).MaxHeatCoilFluidFlow = CoilMaxVolFlowRate * rho;
+                    if (PTUnit(PTUnitNum).ACHeatCoilType_Num == Coil_HeatingWater) {
+                        SimulateWaterCoilComponents(PTUnit(PTUnitNum).ACHeatCoilName, FirstHVACIteration, PTUnit(PTUnitNum).ACHeatCoilIndex);
+                        CoilMaxVolFlowRate = GetCoilMaxWaterFlowRate("Coil:Heating:Water", PTUnit(PTUnitNum).ACHeatCoilName, ErrorsFound);
+                        if (CoilMaxVolFlowRate != AutoSize) {
+                            rho = GetDensityGlycol(PlantLoop(PTUnit(PTUnitNum).HeatCoilLoopNum).FluidName,
+                                                   DataGlobals::HWInitConvTemp,
+                                                   PlantLoop(PTUnit(PTUnitNum).HeatCoilLoopNum).FluidIndex,
+                                                   RoutineNameSpace);
+                            PTUnit(PTUnitNum).MaxHeatCoilFluidFlow = CoilMaxVolFlowRate * rho;
+                        }
+                    } else if (PTUnit(PTUnitNum).ACHeatCoilType_Num == Coil_HeatingSteam) {
+                        SimulateSteamCoilComponents(PTUnit(PTUnitNum).ACHeatCoilName,
```

Comment:
Does calling Simulate here have any side effects?  It seems like you don't really need to simulate, just make sure input is processed and mine out the max steam flow rate, but I understand that calling Simulate is generally the safest way to ensure everything has been processed.

Reply:
Agreed, might be a better way to get the info, it's just what was here. So it's open.